README.md

```diff
@@ -1,4 +1,56 @@
 # scikit-stats: statistics tools and utilities
 
+[![Build Status](https://dev.azure.com/matthieumarinangeli/matthieumarinangeli/_apis/build/status/scikit-hep.scikit-stats?branchName=master)](https://dev.azure.com/matthieumarinangeli/matthieumarinangeli/_build/latest?definitionId=3&branchName=master)
```

Comment:
At some point in the future we can use the scikit-hep account? No rush, of course, as having the CI via your account works just fine :-).

Reply:
@eduardo-rodrigues agreed !